README.md

```diff
@@ -2,10 +2,10 @@
   <img alt="Graphwise Logo" src=".github/Graphwise_Logo.jpg">
 </p>
 
-# Talk to Your Graph (TTYG) Evaluation
+# QA Evaluation
 
-TTYG Evaluation is a Python module for evaluating whether LLM agents correctly orchestrate and invoke available 
-tools to answer user questions, based on a gold-standard corpus of tool call expectations.
+This is a Python module for assessing the quality of question-answering systems such as ones based on LLM agents, based on a set of questions and reference answers for them. This includes evaluating the final answer and the steps used
+to reach the answer (such as orchestrated and invoked tools), compared to the given reference steps.
```

Comment:
There are still references to the term `gold standard corpus`:
- in the README 
- test files names
Is it possible to do 
``grep -nr "gold" .`` and remove them?

Reply:
Thanks. Replaced!